decimal-counter/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Decimal Counter</title>
+</head>
+
+<body>
+
+  <script>
+
+    function decimalPlaceCounter(stringNumber) {
+      for (var i = 0; i < stringNumber.length; i++) { //loop through the stringNumber to find decimal
+        var str = stringNumber.toString(); //convert number to string
+        var result = str.lastIndexOf("."); // returns the position of last occurrence of value in string
```

Comment:
This does work, however, you are calling the `lastIndexOf` method multiple times in the loop and the value never changes.  So you don't need to put it in the loop, just once works.